src/__tests__/adapters/firebase.test.ts

```
@@ -0,0 +1,285 @@
+    });
+
+    it('should use existing app if already initialized', async () => {
+      mockGetApps.mockReturnValue([mockApp] as any);
+
+      await firebaseAdapter.findUserByEmail('test@example.com');
+
+      expect(mockInitializeApp).not.toHaveBeenCalled();
+    });
+
+    it('should use default collection name if not specified', async () => {
+      delete process.env.FIREBASE_AUTH_COLLECTION;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)